DESCRIPTION

```diff
@@ -16,19 +17,22 @@ License: GPL (>= 2)
 ByteCompile: true
 Depends: 
   R (>= 3.6.0)
-Imports: 
-  Rserve (>= 1.7.3),
-  R6 (>= 2.4.0),
-  uuid (>= 0.1-2),
-  checkmate (>= 1.9.4),
-  mime (>= 0.7),
+Imports:
   Rcpp (>= 1.0.2),
-  jsonlite (>= 1.6)
+  Rserve (>= 1.7.3),
+  R6,
+  uuid,
+  checkmate,
+  mime,
+  jsonlite
```

Comment:
I think having version is useful

Reply:
I think we must keep only critical points.